test/system/spor/TEST_LIB.py

```diff
@@ -166,6 +166,8 @@ def get_array_capacity(arrayId=0):
 
 def kill_process(procname, sig=9):
     for proc in psutil.process_iter():
+        if "irq" in proc.name():
```

Comment:
처음 시작할 때 fio 이름을 갖는 process를 kill 하도록 하였으나 vfio~~라는 이름의 기본 프로세스가 있어 hang이 발생하기에 이를 회피하고자 추가하였음
(걔가 irq라는 키워드를 가지고 있음)

Reply:
오.. 그렇군요 사소하지만 말씀주신 내용을 comment 로 남겨두는게 좋지 않을까 생각이 드는데요 (e.g., # we are not killing the process by "fio" since another similarly-named process like "vfio" could have been running), 아니면 명시적으로 fio는 죽이고 vfio는 안죽인다는 조건을 주는 건 어떨까요??? 
```python
for proc in psutil.process_iter():
  if "vfio" not in proc.name() and "fio" in proc.name():
```
같은 식으로요

Reply:
주석 달아두도록 하겠습니다 감사합니다